### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by Kaskyi**

### CODE REVIEW

File: src/app/modules/xm-sidebar/user/user.component.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import { ChangeDetectionStrategy, Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { matExpansionAnimations } from '@angular/material';
+import { ActivationEnd, Router } from '@angular/router';
+import * as _ from 'lodash';
```
Comment (Zabavsky) [inline src/app/modules/xm-sidebar/user/user.component.ts]: what for you import * from lodash? maybe it is wise to import just forEach from lodash module?

Comment (Kaskyi) [reply src/app/modules/xm-sidebar/user/user.component.ts]: lets create issue to refactor all lodash imports

**Comment by ishkurko** [comment]
@Kaskyi 
Could you provide information, how to switch between sidebars?

**Comment by Kaskyi** [comment]
How to switch to the new sidebar:
First, log in and navigate to `/configuration/specification-management/ui`;
Second, add the next configuration:
```
layoutSidebar: 
    selector: 'xm-widget-sidebar'
```
In conclusion, reload the page.

**Comment by Kaskyi** [comment]
feature/sidebar branch was merged into develop.

**Pull request closed without merging**